src/Microsoft.AspNetCore.Server.Kestrel/KestrelServer.cs

```diff
@@ -156,7 +156,7 @@ public void Start<TContext>(IHttpApplication<TContext> application)
                             }
                             else
                             {
-                                _logger.LogWarning(0, ex, $"Unable to bind to {parsedAddress.ToString()} on the IPv4 loopback interface.");
+                                _logger.LogInformation(0, $"Unable to bind to {parsedAddress.ToString()} on the IPv4 loopback interface.");
```

Comment:
How does shared hosting make the error any different? It's fatal or it isn't.
Yes we know Azure has one specific issue, but overall we've decided that only failing on one protocol isn't fatal.